shell/platform/embedder/tests/embedder_unittests.cc

```diff
@@ -473,6 +473,34 @@ TEST_F(EmbedderTest, VMShutsDownWhenNoEnginesInProcess) {
   }
 }
 
+//------------------------------------------------------------------------------
+///
+TEST_F(EmbedderTest, DartEntrypointArgs) {
+  auto& context = GetEmbedderContext(ContextType::kSoftwareContext);
+  EmbedderConfigBuilder builder(context);
+  builder.SetSoftwareRendererConfig();
+  builder.GetProjectArgs().dart_entrypoint_argc = 2;
```

Comment:
See the pattern used for command line args where a vector is built up in `EmbedderConfigBuilder`. This moves the responsibility of ownership of the args to `EmbedderConfigBuilder` and will be less prone to memory issues in future tests.